src/conversion.ts

```diff
@@ -51,13 +51,30 @@ export function centOffsetToFrequency(offset: number, baseFrequency = 440) {
 const MIDI_NOTE_NUMBER_OF_A4 = 69;
 /**
  * Convert MIDI note number to frequency.
- * @param index MIDI note number.
+ * @param index MIDI note number or MTS value.
  * @returns Frequency in Hertz.
  */
 export function mtof(index: number) {
   return 440 * Math.pow(2, (index - MIDI_NOTE_NUMBER_OF_A4) / 12);
 }
 
+function mtsRound(mtsValue: number): number {
+  return Math.round(mtsValue * 100000) / 100000;
```

Comment:
How critical is it that this result is stable? Javascript's number type cannot represent decimals exactly under the hood.

Reply:
I think not very much. Currently there's no use case in SW for the mtsValue to be 128 or higher, but at the most, if for some reason someone used it in an MPE style tuning the upper bound would be 1920 or 2048. It only needs two-places of precision too.
I originally wanted to round to closest possible value in the three-byte representation which is why I made it a separate method. Decided it wasn't worth the debugging for now, but I kept the abstraction in.